### PR TITLE
Move from pandas plot method to matplotlib exclusively in chapter 9

### PR DESCRIPTION
**Pull request by jelletreep**
#92

### CODE REVIEW

File: book/data-science-with-pandas-4.ipynb [hunk 1]
```diff
@@ -263,25 +265,26 @@
...
+    "ax.set_xlabel('Hindfoot length [mm]', fontsize=14)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "02919834-6764-473f-b497-114ec861ef5f",
    "metadata": {},
    "source": [
-    "This time we added some new parameters to our call: \n",
+    "This time we first created an 'axes' object  on which we add the plot and other customizations to our plot: \n",
```
Comment (StefanoRapisarda) [inline book/data-science-with-pandas-4.ipynb:281]: --> This time we first created a Figure ('''fig''') and an Axes ('''ax''') object. On the Axes object we add the plot and specify some customizations:

**Review by StefanoRapisarda — CHANGES_REQUESTED** [review]
Just small change in one of the code descriptions

File: book/data-science-with-pandas-4.ipynb (hunk 1 above)
Comment (jelletreep) [reply book/data-science-with-pandas-4.ipynb]: Thank you @StefanoRapisarda! I included your suggestion!

**Comment by jelletreep** [comment]
@StefanoRapisarda, Thank you! I included your suggestion

**Review by StefanoRapisarda — APPROVED** [review]

**Pull request merged**